pages.de/common/conda.md

```diff
@@ -0,0 +1,37 @@
+# conda
+
+> Ein Kommandozeilentool und Paketverwaltung für beliebige Programmiersprachen.
+> Manche Unterbefehle (z.B. `conda create`) besitzen ihre eigene Dokumentation.
```

Comment:
We have a template for this: <https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#de>
```suggestion
> Manche Unterbefehle wie `conda create` sind separat dokumentiert.
```